storagenode/storagenodedb/database.go

```diff
@@ -27,13 +28,14 @@ type Config struct {
 
 // DB contains access to different database tables
 type DB struct {
+	logger   *zap.Logger
```

Comment:
use log as the name, that way the logging lines look nicer.

Reply:
done